src/geo/transform.js

```diff
@@ -2075,11 +2109,23 @@ class Transform {
         }
     }
 
-    getCameraToCenterDistance(projection: Projection): number {
-        const t = getProjectionInterpolationT(projection, this.zoom, this.width, this.height, 1024);
+    getCameraToCenterDistance(projection: Projection, zoom: ?number): number {
+        zoom = zoom || this.zoom;
```

Comment:
Nit: it's more idiomatic to use default params for things like this (and you can use `this` there too)